scst-scan/setup-supply-chain-component.md

```diff
@@ -0,0 +1,120 @@
+# Setup the Supply Chain Component
+
+This topic covers how to install the Trivy Supply Chain Component, create custom Supply Chain Component using the SCST - Scan 2.0, and view the components that are available to be used in the Tanzu Supply Chain.
+
+## <a id="overview"></a> Overview
+
+* [Install Trivy Supply Chain Component](./setup-supply-chain-component.md#install-trivy-supply-chain-component)
```

Comment:
**Issue**: double check your ID links for all 3 links

Reply:
In Visual Studio I tried confirming this. For links within the same page, it doesn't seem to take me to the section. I did confirm I used auto completion to specify these links and these are what Visual studio suggested to me. How can I confirm these links work?

Reply:
Oh, for this I go to https://github.com/pivotal/docs-tap/blob/dc877e214dcfc18ea14d347b8f09ea7e071d588c/scst-scan/setup-supply-chain-component.md (the preview in github) and then try clicking on the links...

i'm not sure which is a more accurate representation tbh.

Reply:
I tried both ways but it still doesn't render anything when I click on the links. But in the id tags and the link I matched it string to string I believe. 

How about we try to merge this and see if docs staging renders this?

This is slightly digressing but from Denny's comment, these paragraph sections needs to show up on the right side bar like here 
<img width="1828" alt="Screenshot 2024-02-12 at 1 12 23 PM" src="https://github.com/pivotal/docs-tap/assets/7121833/b0f33428-490b-4e8e-8c2c-cd86f3ee4fd0">
But I don't know how to test for that besides just merging this. So that's another reason to try merge this and see what renders?